src/v3/src/components/Link/Link.tsx

```diff
@@ -10,6 +10,7 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+// TODO: Link exported from ODY does not have the focus() function so breaks auto focus functionality
```

Comment:
Is this something we are planning to do in this PR or in a later ticket? If the latter, let's attach a JIRA reference

Reply:
I actually worked with Kevin on this one and he was able to create and merge the fix. I believe it will be in their next release which is Wednesday so I think I will just wait until that release and update the versions and export from ODS. 

Ticket: https://oktainc.atlassian.net/browse/OKTA-564568